test/testcafe/spec/EnrollProfileView_spec.js

```diff
@@ -204,23 +194,22 @@ test.requestHooks(requestLogger, EnrollProfileSignUpAllBaseAttributesMock)('All
     division: 'Division',
     department: 'Department',
     managerId: 'Manager ID',
-    manager: 'Manager',
+    //manager: 'Manager', //TODO: Found multiple elements with the text of: /Manager/
   };
 
-  Object.keys(formFieldToLabel).forEach(async (formField) => {
+  for (const formFieldLabel of Object.values(formFieldToLabel)) {
     // verify all base attributes map to correct translation
     // all 'label' fields for base attributes in json are appended with a '1'
-    await t.expect(await enrollProfilePage.form.fieldByLabelExists(formFieldToLabel[formField])).eql(true);
-  });
+    await t.expect(enrollProfilePage.form.fieldByLabelExists(formFieldLabel)).eql(true);
```

Comment:
I think we want `.ok()` here as well, `form.fieldByLabelExists` uses `.exists` under the hood

Reply:
`eql(true)` and `ok()` are almost same.
They both support [TestCafe's smart assertion](https://testcafe.io/documentation/402837/guides/basic-guides/assertions#smart-assertion-query-mechanism), so both should wait for `.exists`
`eql(true)` is just more strict (like `===` vs `==`)

Reply:
I'm personally in favor of using `.eql(true)` instead of `.ok()` but we just want to be consistent